assets/js/components/Table/Table.test.jsx

```diff
@@ -119,5 +119,24 @@ describe('Table component', () => {
         expect(table.querySelectorAll('tbody > tr')).toHaveLength(1);
       });
     });
+
+    it('should return empty state message when data is empty', async () => {
```

Comment:
Why do you need `async` here? I don't see any `await` or any async operation

Reply:
You are fully right! i will remove it.